### DISCUSSION TITLE
How to dismiss keyboard in flutter without back key

### DISCUSSION THREAD

**Discussion by amarguptas** [community/community#29895] [Mobile] [answered]
How to dismiss keyboard in flutter without back key

**Comment by surajadkhari** [comment, marked as answer]
Wrap your whole parent widget most commonly used is the Scaffold widget or SafeArea with the GestureDetector.

Inside its onTap property, we need to pass unfocus function, which removes the focus from the current node and moves its primary focus to another node.
For more check out :https://medium.com/@surajadkhari/flutter-keyboard-dismiss-in-textformfield-and-onscroll-e35f6cbc5dd1

![medium](https://user-images.githubusercontent.com/48079501/185310114-1f4db1bb-5e8e-4a08-87a7-033a234fda66.jpg)

**Reply by amarguptas** [reply]
I got it thank you
